app/src/main/AndroidManifest.xml

```diff
@@ -124,7 +124,17 @@
             </intent-filter>
         </receiver>
 
+        <receiver android:name="org.matrix.matrixandroidsdk.car.CarBroadcastReceiver">
+            <intent-filter>
+                <action android:name="org.matrix.matrixandroidsdk.ACTION_MESSAGE_HEARD" />
+            </intent-filter>
+            <intent-filter>
+                <action android:name="org.matrix.matrixandroidsdk.ACTION_MESSAGE_REPLY" />
+            </intent-filter>
```

Comment:
Nitpick: Both of these actions can be contained in a single ``<intent-filter>``, and in this case would make your intention clearer.